components/ai_chat/resources/custom_site_distiller_scripts/scripts/github.com/distiller.ts

```diff
@@ -7,13 +7,13 @@ import { LEO_DISTILLATION_LEVEL } from '../distillation'
 import { distillBranches } from './branches'
 import { GetPageType, SupportedPage } from './utils'
 
-let _DISTILLATION_LEVEL = LEO_DISTILLATION_LEVEL.LOW
+let _DISTILLATION_LEVEL = LEO_DISTILLATION_LEVEL.FULL
```

Comment:
Distillation levels presently aren't utilized very extensively throughout the distillation scripts. For now, we can default to full distillation, and evaluate whether or not we have reasons to pull it back in the future.